### PR TITLE
Feature/ipfs-lib-migration

### PR DESCRIPTION
**Pull request by Sorizen**

### CODE REVIEW

File: packages/sdk/package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
     "@rollup/plugin-terser": "^0.4.0",
     "@rollup/plugin-typescript": "^11.0.0",
     "@swc/cli": "^0.1.62",
-    "@swc/core": "^1.3.53",
+    "@swc/core": "1.3.53",
     "@swc/jest": "^0.2.26",
```
Comment (lukachi) [inline packages/sdk/package.json:87]: remove?

File: packages/vue-web3-provider/rollup.config.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export default {
     nodePolyfills(),
     alias({
       entries: [
-        {find: 'ipfs-http-client', replacement: '../../node_modules/ipfs-http-client/dist/index.min.js'},
+        {find: 'kubo-rpc-client', replacement: '../../node_modules/kubo-rpc-client/dist/index.min.js'},
```
Comment (lukachi) [inline packages/vue-web3-provider/rollup.config.js:30]: ?

File: packages/sdk/rollup.config.js [hunk 3]
```diff
@@ -39,7 +39,6 @@ export default {
     nodePolyfills(),
     alias({
       entries: [
-        {find: 'ipfs-http-client', replacement: '../../node_modules/ipfs-http-client/dist/index.min.js'},
```
Comment (lukachi) [inline packages/sdk/rollup.config.js:42]: ?

Comment (Sorizen) [reply packages/sdk/rollup.config.js:42]: done

File: packages/sdk/package.json (hunk 1 above)
Comment (Sorizen) [reply packages/sdk/package.json]: done

**Review by lukachi — APPROVED** [review]

**Pull request merged**